tools/labs/templates/kernel_profiling/0-demo/io-app.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * lab008 - sync disk writes to a file, with (syscall) latency outliers.
+ *
+ * 21-May-2015	Brendan Gregg	Created this.
```

Comment:
lab008? If this is from the original then it is fine, otherwise we could just add: ``io-app.c - ...``

Reply:
It's from the original.